.azure/pipelines/ci.yml

```diff
@@ -10,36 +10,13 @@ trigger:
     - master
     - release/*
     - internal/release/*
-  paths:
-    exclude:
-    - .github/*
-    - .vscode/*
-    # Exclude only a few specific Markdown files because some of them ship.
-    - docs/*
-    - CODE-OF-CONDUCT.md
-    - CONTRIBUTING.md
-    - LICENSE.TXT
-    - README.md
-    - SECURITY.md
-    # Do not ignore the THIRD-PARTY-NOTICES.TXT file because it ships.
```

Comment:
Decided these exclusions were a mistake. Though the product won't change when these files are updated, we should keep our builds up-to-date.